spec/controllers/comments_controller_spec.rb

```diff
@@ -4,8 +4,7 @@
   describe "POST /" do
     it "responds with 200" do
       post :create, params: { comment: { message: "Hello, world!", user_id: 1}, post_id: 1 }
-      @post = Post.find(1)
-      expect(response).to redirect_to(post_path(@post)) # This test is passing but needs to be changed
+      expect(response).to redirect_to(post_path(Post.find(1)))
     end
 
```

Comment:
Does this test fail due to hard coded user_id ?

Reply:
No it passes. We probably should have created a login step before this so we'd have a session user but this worked so we left it. We'll probably revisit the controller tests at some point because we haven't got our head round them completely and they could be improved.